code/modules/client/preferences_savefile.dm

```diff
@@ -219,7 +218,11 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	be_special		= SANITIZE_LIST(be_special)
 	pda_style		= sanitize_inlist(pda_style, GLOB.pda_styles, initial(pda_style))
 	pda_color		= sanitize_hexcolor(pda_color, 6, 1, initial(pda_color))
-	key_bindings 	= sanitize_islist(key_bindings, deepCopyList(GLOB.keybinding_list_by_key)) 
+	key_bindings 	= sanitize_islist(key_bindings, list())
+
+	if(!length(key_bindings))
```

Comment:
```suggestion
	if(!length(key_bindings))
		key_bindings = (hotkeys) ? deepCopyList(GLOB.hotkey_keybinding_list_by_key) : deepCopyList(GLOB.classic_keybinding_list_by_key)
		parent.update_movement_keys()
```
In case `load_default_keybindings` doesn't happen. It's not terribly reliable.

Reply:
better to be safe